src/apps/statements/service/storeStatements/index.ts

```diff
@@ -33,20 +34,25 @@ const cloneAttachments = (attachmentModels: AttachmentModel[]): AttachmentModel[
 };
 
 export default (config: Config) => {
-  return async (opts: StoreStatementsOptions): Promise<string[]> => {
-    checkScopes(STATEMENT_WRITE_SCOPES, opts.client.scopes);
-    const preValidatedModels = preValidationSetup(config, opts.models);
+  return async ({
+    models,
+    attachments,
+    client,
+    priority = StatementProcessingPriority.MEDIUM,
+  }: StoreStatementsOptions): Promise<string[]> => {
```

Comment:
NIT: Technically, we prefer the [opts pattern](https://learninglocker.atlassian.net/wiki/spaces/LL/pages/1057685522/Too+Many+Function+Parameters) because it's clearer in the function body what has been passed through to the function vs what is internal to the function.

Reply:
@ryansmith94 , yeah, I didn't want to do this but I needed to give the default value only to `priority` because too many places(mostly tests) were yelling at me 👀  In case of `opts` param I would have been needed to assign the default value to a whole `opts` or add an additional code to the function body

Reply:
Makes sense 👍